README.md

```diff
@@ -5,16 +5,25 @@ This is a deck building app for Star Wars: The Card Game.  This project is inspi
 
 ## Hacking on this Project
 ### You need
-- Ruby 3.2
-- Rails 7
-- Postgres 14
+
+1. For local deployment
+    - Ruby 3.2
+    - Rails 7
+    - Postgres 14
+2. For docker
```

Comment:
Maybe `For container deployment` here to avoid the redundancy with the next line?

Reply:
fixed